tests/test_console.py

```diff
@@ -74,13 +77,18 @@ def test_send_data_no_endpoint(self):
 Postgresql = testing.postgresql.PostgresqlFactory(cache_initialized_db=True)
 
 
+def get_test_data():
+    SITE_ROOT = os.path.realpath(os.path.dirname(__file__))
+    json_url = os.path.join(SITE_ROOT, 'test_data', 'test_response_1.json')
+    responses_json = []
+    with open(json_url) as json_data:
```

Comment:
When I use this the submit_test_responses function fails because it's expecting a list of json responses to do a for loop over. So when there's no list it loops through the contents of the response.

I want to keep the for loop in though so in the future multiple json files can be used as test data.